src/components/file-input/file-input.style.tsx

```diff
@@ -0,0 +1,91 @@
+import styled, { css } from "styled-components";
+import type { InputProps } from "../../__internal__/input";
+import type { ValidationProps } from "../../__internal__/validations";
+import StyledTypography from "../typography/typography.style";
+
+export const StyledHintText = styled.div`
+  margin-top: 0px;
+  margin-bottom: 8px;
+  color: var(--colorsUtilityYin055);
+  font-size: 14px;
+`;
+
+export const ErrorBorder = styled.span`
+  position: absolute;
+  z-index: 6;
```

Comment:
question: what is this z-index used for?

It stood out because 6 seems to be a bit random to me, but I've tried removing it on storybook and it doesn't make a difference.

Reply:
Excellent question that I can't answer 😅 I'll hold my hand up and admit this was copy-pasted from an equivalent styled component elsewhere (almost certainly Textbox but I don't specifically remember). Similar to what @edleeks87  commented on the hint text, that could arguably be moved to a central location (but as I argued there I'd rather not do it in this PR). Most likely the z-index can be removed from the other ones too, or maybe it's specific to that component that I copied it from and just not needed here?

For now I think I'll keep things separate and just remove the z-index from this component after confirming it's not needed. Thanks for spotting it.